plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -694,6 +696,63 @@ public void testTimestampColumnAndTimestampWithTimeZoneConstant()
         }
     }
 
+    @Test
+    public void testAccessDeniedToTableWithExtraCredentialImpersonation()
+            throws Exception
+    {
+        String user = "test_access_denied_to_table_" + randomTableSuffix();
+        Session userSession = Session.builder(getSession())
```

Comment:
maybe underline `test_access_denied_to_table` via renaming `Session` to `deniedSelectSession?